README.md

```diff
@@ -330,6 +333,94 @@ RUN go build -o hello hello.go && \
     xx-verify hello
 ```
 
+## Rust
+
+Building Rust can be achieved with the `xx-cargo` wrapper that automatically
+sets up the target triple and also `pkg-config` and C compiler.
+
+The wrapper supports rust installed via [`rustup`](https://rustup.rs/)
+(alpine/debian), distribution packages (alpine/debian) and the [official `rust` image](https://hub.docker.com/_/rust).
+
+### Building on Alpine
+
+```dockerfile
+# official rust image
+FROM --platform=$BUILDPLATFORM rust:alpine
+RUN apk add clang lld
+# ...
+ARG TARGETPLATFORM
+RUN xx-apk add xx-c-essentials
```

Comment:
This is needed on `rust` image?

Reply:
Will take another look but `musl-dev` not installed for alpine base image: https://github.com/rust-lang/docker-rust/blob/47a11a42c5a321d688fae525eaa7f1cf9940198b/1.64.0/alpine3.16/Dockerfile#L5

For debian one I'm not sure: https://github.com/rust-lang/docker-rust/blob/47a11a42c5a321d688fae525eaa7f1cf9940198b/1.64.0/bullseye/Dockerfile. It's based on `buildpack-deps:bullseye` which might already have theses packages.

Reply:
>  but musl-dev not installed for alpine base image

If it is not needed without cross-compile then why is it needed for us

Reply:
Ok this is not required with rustup or rust image.